workshop1/slides/src/solarized.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
This file is not strictly needed, it was used to generate the template file.

Reply:
Ok will remove